src/server/db_slice.h

```diff
@@ -468,8 +469,12 @@ class DbSlice {
     }
   };
 
-  // the table that maps keys to the clients that are tracking them.
-  absl::flat_hash_map<std::string, absl::flat_hash_set<facade::Connection::WeakRef, Hash>>
+  using AllocatorType = PMR_NS::polymorphic_allocator<
+      std::pair<std::string, absl::flat_hash_set<facade::Connection::WeakRef, Hash>>>;
+
+  absl::flat_hash_map<std::string, absl::flat_hash_set<facade::Connection::WeakRef, Hash>,
+                      absl::container_internal::hash_default_hash<std::string>,
```

Comment:
this is the default hash used by this absl::flat_hash_map. it won't compile without this default hash (this hash supports taking string_view as input parameter for [] operator, for example.)

Reply:
Yeah, I agree that it's not looking good, but it doesn't seem like there's any other way to add that last template parameter. I wonder why the default parameters are in internal namespace, that's just odd..

Reply:
> Yeah, I agree that it's not looking good, but it doesn't seem like there's any other way to add that last template parameter. I wonder why the default parameters are in internal namespace, that's just odd..

It's annoying, I once needed it and had to use templates to avoid specifying it